x/shared/helpers/service_test.go

```diff
@@ -1,78 +1,181 @@
 package helpers
 
-import "testing"
+import (
+	"testing"
+
+	sharedtypes "github.com/pokt-network/poktroll/x/shared/types"
+)
+
+func TestIsValidService(t *testing.T) {
+	tests := []struct {
+		testCase string
+		id       string
+		name     string
+		expected bool
+	}{
+		{
+			testCase: "Valid ID and Name",
+			id:       "Service1",
+			name:     "Valid Service Name",
+			expected: true,
+		},
+		{
+			testCase: "Valid ID and empty Name",
+			id:       "Srv",
+			name:     "", // Valid because the service name can be empty
+			expected: true,
+		},
+		{
+			testCase: "ID exceeds max length",
+			id:       "TooLongId123", // Exceeds maxServiceIdLength
+			name:     "Valid Name",
+			expected: false,
+		},
+		{
+			testCase: "Name exceeds max length",
+			id:       "ValidID",
+			name:     "This service name is way too long to be considered valid since it exceeds the max length",
+			expected: false,
+		},
+		{
+			testCase: "Empty ID",
```

Comment:
Wdyt about explicitly stating the expectation in the subtest name, like the others do:

```suggestion
			testCase: "Empty ID is invalid",
```